CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Nothing yet!
+### Added
+
+- Add support for utility prefixes in v4 ([#14501](https://github.com/tailwindlabs/tailwindcss/pull/14501))
```

Comment:
This is mega nit but now that the prefix is not longer only in front of the utility, maybe should maybe not call it that. Also I think the version identifier is already in the changelog so might as well drop that:

```suggestion
- Add support for prefixes ([#14501](https://github.com/tailwindlabs/tailwindcss/pull/14501))
```